documentation/projects/proposals/publish_dataset/20230706-project_proposal.md

```diff
@@ -0,0 +1,207 @@
+# Project Proposal: Providing and maintaining an Openverse image dataset
+
+**Author**: @zackkrida
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @AetherUnbound
+- [ ] @sarayourfriend
+
+## Project summary
+
+This project aims to publish and regularly update a dataset or _datasets_ of
+Openverse' media metadata. This project will provide access to the data
+currently served by our API, but which is difficult to access in full, requires
+significant time, money, and compute resources.
+
+## Motivation
+
+For this project, understanding "why?" we would do this is of paramount
+importance. There are several key philosophical and technical advantages to
+sharing this dataset with the public.
+
+### Open Access + Scraping Prevention
+
+Openverse is and always has been, since its days as
+["CC Search"](https://creativecommons.org/2021/12/13/dear-users-of-cc-search-welcome-to-openverse/),
+informed by principles of the open access movement. Openverse strives to remove
+all all financial, technical, and legal barriers to accessing the works of the
+global commons.
+
+Due to technical and logistical limitations, we have previously been unable to
+accessibly provide access to the full Openverse dataset. Today, users need to
+invest significant time and money into scraping the Openverse API in order to
+access this data. These financial and technical barriers to our users are deeply
+inequitable. Additionally, this scraping disrupts Openverse access and stability
+for all users. It also requires significant maintainer effort to identify,
+mitigate, and block scraping traffic.
+
+By sharing this data as a full dataset on
+[HuggingFace](https://huggingface.co/datasets), we can remove these barriers
+allow folks to access the data provided by Openverse.org and the Openverse API
+without restriction.
+
+### Contributions Back to Openverse
+
+Easily accessed Openverse datasets will facilitate easier generation of machine
+labels, translations, and other supplemental data which can be used to improve
+the experience of Openverse.org and the API. This data is typically generated as
+part of the
+[data preprocessing](https://huggingface.co/docs/transformers/preprocessing)
+stage of model training.
+
+HuggingFace in particular will enable community members to analyze the dataset
```

Comment:
This wording feels to me like this is an obligation HuggingFace has after the fact. Would you be comfortable with this slight rewarding so it better conveys a sense that this is what HuggingFace is currently capable of?

```suggestion
Presence on HuggingFace in particular will enable community members to analyze the dataset
```